### PR TITLE
Add File()

### PR DESCRIPTION
**Pull request by 6543**
similar to EnvVar()

It's just a smal helper to make `NewValueSourceChain` more usefull :)

### CODE REVIEW

**Comment by 6543** [comment]
will be used in https://github.com/woodpecker-ci/woodpecker/pull/2951 ...

**Review by bartekpacia — APPROVED** [review]
This LGTM, but I'm holding off for more maintainers to speak up :)

**Review by Juneezee — APPROVED** [review]
Makes sense and LGTM

**Pull request merged**
